src/Platform/Microsoft.Testing.Extensions.AzureDevOpsReport/AzureDevOpsReporter.cs

```diff
@@ -292,7 +294,11 @@ private async Task WriteExceptionAsync(string? explanation, Exception? exception
 
             string err = AzDoEscaper.Escape(message);
 
-            string line = $"##vso[task.logissue type={severity};sourcepath={relativeNormalizedPath};linenumber={location.Value.LineNumber};columnnumber=1]{err}";
+            string formattedMessage = RoslynString.IsNullOrEmpty(testDisplayName)
+                ? err
+                : $"[{AzDoEscaper.Escape(testDisplayName)}] {err}";
```

Comment:
While the `[]` and space here don't need escaping, I wonder if it's better to construct the whole message we want at once, and then just escape once.

What we have so far is we escape two parts of it and do concatenation.